pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -368,6 +371,23 @@ def replay_genargs_handle_argument(arg: str) -> str | None:
     return arg
 
 
+def calculate_exports_flag(line: list[str], export_all: bool) -> str:
+    objects = [arg for arg in line if arg.endswith(".a") or arg.endswith(".o")]
+    PYODIDE_ROOT = common.get_pyodide_root()
+    result = subprocess.run(
+        [PYODIDE_ROOT / "emsdk/emsdk/upstream/bin/llvm-nm", "-j", "--export-symbols"]
```

Comment:
I think we can use `emnm`. It is just a wrapper of `llvm-nm`, and it is in our `PATH` like `emcc`, etc.

https://github.com/emscripten-core/emscripten/blob/main/tools/emnm.py

Reply:
Good find, I hadn't noticed that one.